src/routes/chains/chains.controller.ts

```diff
@@ -20,7 +19,7 @@ import { AboutChain } from './entities/about-chain.entity';
 export class ChainsController {
   constructor(private readonly chainsService: ChainsService) {}
 
-  @ApiImplicitQuery({
+  @ApiQuery({
```

Comment:
As per the [breaking changes](https://docs.nestjs.com/openapi/migration-guide#breaking-changes), `@ApiImplicitQuery` is now `@ApiQuery`.